it.hjson

```diff
@@ -14,39 +14,39 @@
                         Generic - Used for multiple things
      --------------------------------------------------------------------- */
   "generic.callbacks": {
-    "userBannedBy": "<a href=\"tg://user?id=%s\">Questo utente</a><b> è stato bandito da </b>%s"
+    "userBannedBy": "<a href=\"tg://user?id=%s\">Questo utente</a><b> è stato bannato da </b>%s"
     # ℹ️ NOTE: Signatures are used for intelligent text switching. MUST be a uniquely identifiable portion of the full text,
     # and NOT contain any formatting or assume the value of any dynamic values.
-    "userBannedBySignature": " è stato bandito da "
-    "userBannedAnswerText": "Utente escluso"
+    "userBannedBySignature": " è stato bannato da "
+    "userBannedAnswerText": "Utente bannato"
     # ℹ️ NOTE: Signatures are used for intelligent text switching. MUST be a uniquely identifiable portion of the full text,
     # and NOT contain any formatting or assume the value of any dynamic values.
-    "userUnbannedBy": "<a href=\"tg://user?id=%s\">Questo utente</a><b> non era stato pianificato da </b>%s"
-    "userUnbannedBySignature": " non era stato pianificato da "
-    "userUnbannedAnswerText": "Utente non bandito"
+    "userUnbannedBy": "<a href=\"tg://user?id=%s\">Questo utente</a><b> è stato sbannato da </b>%s"
+    "userUnbannedBySignature": " sbannato da "
+    "userUnbannedAnswerText": "Utente sbannato"
 
     # ℹ️ Example: /JoinVerify enabled by Andy
     "featureEnabledBy": "%s<b> abilitato da </b>%s"
     "featureEnabledAnswerText": "%s abilitato"
     "featureAlreadyEnabled": "%s<b> è già abilitato</b>"
-    "featureAlreadyEnabledAnswerText": "%s Già abilitato"
+    "featureAlreadyEnabledAnswerText": "%s è già stato abilitato"
     "featureDisabledBy": "%s<b> disabilitato da </b>%s"
-    "featureDisabledAnswerText": "%s Disabilitato"
-    "featureAlreadyDisabled": "%s<b> è già disabilitato</b>"
-    "featureAlreadyDisabledAnswerText": "%s Già disabilitato"
+    "featureDisabledAnswerText": "%s disabilitato"
+    "featureAlreadyDisabled": "%s<b> è già stato disabilitato</b>"
+    "featureAlreadyDisabledAnswerText": "%s è già disabilitato"
 
-    "joinLimitRestrictionsReset": "<a href=\"tg://user?id=%s\">Questo utente</a><b> aveva il loro </b><code>/JoinLimit</code><b> restrizioni ripristinate da </b>%s"
-    "joinLimitRestrictionsResetAnswerText": "Restrizioni Ripristino!"
+    "joinLimitRestrictionsReset": "<a href=\"tg://user?id=%s\">A questo utente</a><b> sono state rimosse le restrizioni di </b><code>/JoinLimit</code> dall'amministratore %s"
+    "joinLimitRestrictionsResetAnswerText": "Rimosse restrizioni."

```

Comment:
Periods on `AnswerText`s seem ugly to me and are not present on the original version

Can we remove them all please?

```suggestion
    "joinLimitRestrictionsResetAnswerText": "Rimosse restrizioni"
```